src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int COEFFICIENT = 10;
+    private static final int NUMBER_OF_FIGURE = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.toString();
+
+    public Figure getRandomFigure() {
+        String color = new ColorSupplier().getRandomColor();
+
+        switch (new Random().nextInt(NUMBER_OF_FIGURE)) {
+            case 0:
+                return new Circle(new Random().nextDouble() * COEFFICIENT, color);
```

Comment:
It would be better to move the creation of each shape into a new method, such as createRandomCircle()